python/cudf/cudf/core/series.py

```diff
@@ -10,7 +10,7 @@
 from typing import Any, Optional
 from uuid import uuid4
 
-import cupy
+import cupy as cp
```

Comment:
changing this import and not changing it's usages in this file will break the code here.